snippets/snip-private-clusters-public-ingress.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following modules:
+//
+// * modules/private-clusters-default.adoc
+// * modules/private-clusters-about.adoc
+// * modules/private-clusters-about-aws.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+If the cluster has any public subnets, LoadBalancer-type services created by administrating users might be publicly accessible. To ensure cluster security, Verify that these services are explicitly annotated as private.
```

Comment:
```suggestion
If you create additional load balancer services after installing a cluster that has any public subnets, the load balancers might be publicly accessible. To ensure cluster security, verify that these services are explicitly annotated as private.
```
Recommending rewording this to use the lower case term of "load balancer". That's how we refer to them in docs like this: https://docs.openshift.com/container-platform/4.11/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer.html 

I would recommend either using the lower case term, or explicitly mentioning `type: LoadBalancer` since that's how it's specified in the service definition. What do you think?

Also a tiny typo fix with "Verify"->"verify".